torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -1616,6 +1621,230 @@ def get_layout(self) -> Layout:
         return self._layout
 
 
+@register_layout(Int4CPULayout)
+class Int4CPUAQTTensorImpl(AQTTensorImpl):
+    """
+    TensorImpl for int4 CPU layout for affine quantized tensor, this is for int4 only,
+    used by tinygemm kernels `_weight_int4pack_mm`
```

Comment:
please update docs to remove tinygemm related things

Reply:
tinygemm kernel is also used in `_weight_int4pack_mm_for_cpu`, I change `_weight_int4pack_mm` to `_weight_int4pack_mm_for_cpu`.

Reply:
oh OK, I thought you are using some other things